lib/Doctrine/Common/DataFixtures/Executor/ORMExecutor.php

```diff
@@ -60,6 +58,11 @@ public function getObjectManager()
         return $this->originalManager;
     }
 
+    public function getEntityManager(): EntityManagerInterface
+    {
+        return $this->em;
+    }
```

Comment:
I honestly dislike that the `ORMExecutor` is leaking the EM now.

Reply:
Didn't spot that, indeed it's not great… one way to avoid it would be to introduce a `wrapInTransaction` proxy method on `ORMExecutor`